Analysis/DataModel/include/AnalysisDataModel/HFSecondaryVertex.h

```diff
@@ -38,6 +39,7 @@ DECLARE_SOA_TABLE(HFSelTrack, "AOD", "HFSELTRACK",
 using BigTracks = soa::Join<Tracks, TracksCov, TracksExtra, HFSelTrack>;
 using BigTracksMC = soa::Join<BigTracks, McTrackLabels>;
 using BigTracksPID = soa::Join<BigTracks, pidRespTPC, pidRespTOF>;
+using V0DataExt = aod::V0DataExt;
```

Comment:
Yes, line 52

Reply:
Well, an alias that is used just once is not a very useful one, is it? :-)

Reply:
It is called `V0Datas` now but my comment still applies. This alias is used only once and you use `aod::V0Datas` everywhere else. So I don't see any benefit of defining this alias at all.

Reply:
I don't think (or know) that it makes any overkill, but ok, since you don't like it, I removed it. I introduced it when I did not know how many times I would have needed it.